DataFormats/TauReco/src/classes_def_2.xml

```diff
@@ -1,19 +1,20 @@
 <lcgdict>
   
-  <class name="reco::PFTau" ClassVersion="18">
+  <class name="reco::PFTau" ClassVersion="19">
+   <version ClassVersion="19" checksum="4003955973"/>
    <version ClassVersion="18" checksum="1318776182"/>
    <version ClassVersion="17" checksum="1523260402"/>
-    <version ClassVersion="16" checksum="2695990650"/>
```

Comment:
please put the old indentation back so that only essential changes appear in diffs